src/shell-app-system.c

```diff
@@ -611,3 +644,9 @@ shell_app_system_get_installed (ShellAppSystem *self)
 {
   return shell_app_cache_get_all (shell_app_cache_get_default ());
 }
+
+gboolean
+shell_app_system_has_starting_apps (ShellAppSystem *self)
```

Comment:
I couldn't find any uses of this function in this tree. https://github.com/endlessm/eos-desktop/blob/ee7659e61a5dcb804065b107f68ca3b81ce1f10c/js/ui/workspaceMonitor.js#L64 is the last reference I can find.

I wonder what it's for?

Reply:
You're right that it's unused. In fact, the app-info-changed signal also is. I think we can drop both.